test/extended/image_ecosystem/s2i_ruby.go

```diff
@@ -19,7 +19,7 @@ import (
 var _ = g.Describe("[image_ecosystem][ruby][Slow] hot deploy for openshift ruby image", func() {
 	defer g.GinkgoRecover()
 	var (
-		railsTemplate = "https://raw.githubusercontent.com/openshift/rails-ex/master/openshift/templates/rails-postgresql.json"
+		railsTemplate = "https://raw.githubusercontent.com/sclorg/rails-ex/master/openshift/templates/rails-postgresql.json"
```

Comment:
would it be better to pull this from openshift/library which we at least sort of control?

Reply:
(or even openshift samples operator's repo, if we're shipping this template?)

Reply:
looks like we are shipping it....in which case you don't even need to get it from the repo, it's available in the cluster (as installed by the samples operator)

Reply:
I debated making that change, but  wavered on the value of some element of upstream verification.

This was sort of a meet in the middle sort of compromise.  No longer the wild wild west but some eye to helping to keep upstream honest.

Thoughts @bparees ?  I'll admit I don't feel super strongly about it though.

Reply:
moving to library probably doesn't have a huge benefit. moving to using the in-cluster template would make the test more robust since it wouldn't be at risk of:

1) github flakes
2) upstream merges that break the test and block the queue

i don't feel that strongly about it though (you guys own the test and will be dealing w/ the fall out if it starts breaking).

Reply:
+1 to using the template on the cluster. In addition to the benefits @bparees mentioned, this lets us catch breaking changes when we bump the samples operator.

Hypothetically we could then create a job that regularly bumps the samples from openshift/library and opens a PR for us to review.